testsuite/integration-arquillian/servers/auth-server/services/testsuite-providers/src/main/java/org/keycloak/testsuite/federation/UserPropertyFileStorage.java

```diff
@@ -111,7 +111,8 @@ private void addCall(String method) {
     public int getUsersCount(RealmModel realm, Map<String, String> params) {
         addCall(COUNT_SEARCH_METHOD);
 
-        return (int) searchForUser(realm, params.get(UserModel.SEARCH), null, null, username -> username.contains(params.get(UserModel.SEARCH))).count();
+        String search = params.get(UserModel.SEARCH);
+        return (int) searchForUser(realm, search, null, null, username -> search == null || username.contains(search)).count();
```

Comment:
This is related to [Quarkus change](https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.18#empty-query-parameter-values-of-type-string). I believe this is now a correct behaviour as previously the test provider worked "by accident".